### ISSUE TITLE
App Icon Design

### ISSUE THREAD

**Issue by alainm23** [alainm23/RetroByte#2]
RetroByte needs a App Icon.

**Comment by ngeshlew** [comment]
@alainm23 I can take a stab at it if you are interested. Please let me know.

**Comment by alainm23** [comment]
@ngeshlew yes please thank you very much, I'm not really sure what the icon could look like, but any idea will be fine for me.

**Comment by ngeshlew** [comment]
@alainm23 you can assign the issue to me.

**Comment by ngeshlew** [comment]
@alainm23 here are the two variations 👇🏽
![IRetroByte](https://github.com/ngeshlew/ngeshlew.github.io/blob/master/opensource/retrobyte.png?raw=true)

Let me know which one you are going with.

**Comment by alainm23** [comment]
@ngeshlew Both designs look amazing but it would be nice to be able to follow the elementary project's design guidelines.
https://elementary.io/es/docs/human-interface-guidelines#iconography

**Comment by ngeshlew** [comment]
@alainm23 I see! Would we say that the designs above are for marketing and campaigns then the one I'm designing next is for the OS, correct?

Also, please let me know which one is your favorite between the two.

Also thanks for the reference to the documentation guide.

**Comment by alainm23** [comment]
@ngeshlew for marketing and campaigns I like the first option, it looks simpler and more elegant.

**Comment by ngeshlew** [comment]
Hey @alainm23, did you have a chance to review the PR?
